rotkehlchen/db/upgrades/v46_v47.py

```diff
@@ -186,24 +186,25 @@ def _remove_nft_collection_assets(write_cursor: 'DBCursor') -> None:
                 identifiers_to_remove,
             )
 
-    @progress_step(description='Reset gemini events and cache')
-    def _reset_gemini_events(write_cursor: 'DBCursor') -> None:
-        write_cursor.execute(
-            'DELETE FROM history_events WHERE location=? AND event_identifier NOT LIKE ?',
-            ((gemini_db_loc := Location.GEMINI.serialize_for_db()), f'{ROTKI_EVENT_PREFIX}_%'),
-        )
-        write_cursor.execute(
-            "DELETE FROM trades WHERE location=? AND link != ''",
-            (gemini_db_loc,),
-        )
-        write_cursor.execute(
-            'DELETE FROM used_query_ranges WHERE name LIKE ? OR name LIKE ? OR name LIKE ? OR name LIKE ?',  # noqa: E501
-            (
-                f'{(gemini_loc := Location.GEMINI.serialize())}_history_events_%',
-                f'{gemini_loc}_trades_%',
-                f'{gemini_loc}_margins_%',
-                f'{gemini_loc}_asset_movements_%',
-            ),
-        )
+    @progress_step(description='Reset exchanges events and cache')
```

Comment:
can't you also put coinbase here? Coinbase has only one extra call which you can just add right? or am I missing something?